qemu/tests/vm_hugetlb_selftests.py

```diff
@@ -0,0 +1,25 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    vm_hugetlb_selftests test
+    1) Execute the hugetlb kernel selftests
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+    kernel_path = params.get("kernel_path", "/home/kernel")
+
+    error_context.base_context("Execute the hugetlb selftests", test.log.info)
+    session.cmd("cd %s" % kernel_path)
```

Comment:
We need to create this path first
` ERROR: Shell command failed: 'cd /home/kernel'    (status: 1,    output: '-bash: cd: /home/kernel: No such file or directory\n')`

Reply:
Sorry, is there a depend patch here?
I don't see where to get the kernel in the guest.

Reply:
@zhenyzha yes, as I said, there's an internal dependent patch, please have a look to the issue as I cannot publish it here :)